### PR TITLE
Developing the Game

### PR DESCRIPTION
**Pull request by SparshaSaha**
Game Development

### CODE REVIEW

File: The Game/Sprites/Bird.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import pygame
+class Bird(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.hitbox = (0, 0, 0, 0)
+        self.frameCount = 0;
+        self.images = ["Sprites/GameImages/birdUp.png", "Sprites/GameImages/birdDown.png"]
+        self.index = 0
```
Comment (sridharswain) [inline The Game/Sprites/Bird.py]: Change "index" to "flipState"

File: The Game/Sprites/CactusDouble.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+class CactusDouble(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.hitbox = (0, 0, 0, 0)
+        self.imageName = "Sprites/GameImages/doubleCactus.png"
+
+    def drawCharacter(self, canvas):
+        canvas.blit(pygame.image.load(self.imageName), (self.x,self.y))
```
Comment (sridharswain) [inline The Game/Sprites/CactusDouble.py]: Load the image once in constructor  and use it for blitting there after.
Well image size is 1k so, not a thing to be bothered

File: The Game/Sprites/Bird.py (hunk 1 above)
Comment (SparshaSaha) [reply The Game/Sprites/Bird.py]: Changed

File: The Game/Sprites/CactusDouble.py (hunk 2 above)
Comment (SparshaSaha) [reply The Game/Sprites/CactusDouble.py]: changed whereever possible

**Review by sridharswain — APPROVED** [review]

**Pull request merged**